scenarios/pubsub_simple.erl

```diff
@@ -0,0 +1,269 @@
+    lager:debug("Client: Response is ~p\n", [Resp]),
+    publish_items_forever(Client, Id, Nodes, ItemId + 1).
+
+work_as_subscriber(Id, Client, Nodes) ->
+    [subscribe(Client, Node) || Node <- Nodes],
+    lager:info("Subscriber, node: ~p~n", [Nodes]),
+    receive_messages_forever(Id, Client).
+
+receive_messages_forever(Id, Client) ->
+    Stanza = escalus:wait_for_stanza(Client, ?WAIT_FOR_STANZA_TIMEOUT),
+    lager:debug("Client: Subscriber ~p got stanza ~p.", [Id, Stanza]),
+    receive_messages_forever(Id, Client).
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% Helpers %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+
+connect_amoc_user(Id) ->
+    connect_amoc_user(Id, <<"res1">>).
+
+connect_amoc_user(Id, Resource) ->
+    Cfg = make_user(Id, Resource),
+    {ok, Client, _} = escalus_connection:start(Cfg),
+    Client.
+
+create_pubsub_nodes(Id) ->
+    pg2:create(?GROUPNAME),
+    pg2:join(?GROUPNAME, self()),
+    Client = connect_amoc_user(1),
+    Nodes = [create_pubsub_node(Client)],
+    provide_nodes_names(Client, Nodes).
+
+create_pubsub_node(Client) ->
+    Node = pubsub_node(),
+    ReqId = id(Client, Node, <<"create">>),
+    NodeConfig = [{<<"pubsub#subscribe">>, <<"1">>},
+                  {<<"pubsub#access_model">>, <<"open">>},
+                  {<<"pubsub#publish_model">>, <<"open">>}],
+    Request = escalus_pubsub_stanza:create_node(Client, ReqId,
+						Node, NodeConfig),
+    escalus:send(Client, Request),
+    {CreateNodeTime, CreateNodeResult} = timer:tc(
+        fun() ->
+            escalus:wait_for_stanza(Client, ?WAIT_FOR_STANZA_TIMEOUT)
+        end),
+
+    case escalus_pred:is_iq_result(CreateNodeResult) of
+        true ->
+            exometer:update(?NODE_CREATE_SUCCESS_COUTER, 1),
+            exometer:update(?NODE_CREATE_TIME, CreateNodeTime);
+        Error ->
+            exometer:update(?NODE_CREATE_FAILS_COUTER, 1),
+            loger:error("Error creating node: ~p", [Error]),
+            exit(connection_failed)
+    end,
+    Node.
+
+pubsub_node() ->
+    Name = pubsub_node_name(),
+    lager:info("CREATED: ~p", [Name]),
+    {?PUBSUB_ADDR, Name}.
+
+rand_name(Prefix) ->
+    Suffix = base64:encode(crypto:strong_rand_bytes(5)),
+    <<Prefix/binary, "_", Suffix/binary>>.
+
+pubsub_node_name() ->
+    Name0 = rand_name(<<"princely_musings">>),
+    re:replace(Name0, "/", "_", [global, {return, binary}]).
+
+id(Client, {NodeAddr, NodeName}, Suffix) ->
+    UserName = escalus_utils:get_username(Client),
+    list_to_binary(io_lib:format("~s-~s-~s-~s",
+				 [UserName, NodeAddr, NodeName, Suffix])).
+
+user_spec(ProfileId, Password, Res) ->
+    [ {username, ProfileId},
+      {server, ?HOST},
+      {host, pick_server()},
```

Comment:
`pick_server()` in current form (on Tide) returns sth like this:
```erlang
[{host, MONGOOSE_IP_ADDR},
 {port, MONGOOSE_PORT}]
```
So this config will not work on Tide currently.